dev-python/poetry-plugin-export/poetry-plugin-export-1.2.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=poetry
+PYTHON_COMPAT=( python3_{8..10} )
+inherit distutils-r1
+
+DESCRIPTION="A plugin that allows the export of locked packages to various formats"
+HOMEPAGE="
+	https://python-poetry.org/
+"
+SRC_URI='https://files.pythonhosted.org/packages/42/4a/b5cafd020ef9294a160950801041d6d0b37ca75db7e8f78f084cae425b47/poetry_plugin_export-1.2.0.tar.gz'
```

Comment:
No hardcoding hashes or version numbers, please. There's got to be a way to get it without that. Let me know if get stuck.

Reply:
I've tried the scheme
`SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz`
which works for version 1.1.2 but not 1.2.0
Encountered a problem as stated here. https://bugs.gentoo.org/580648
The python community indicated that hashless links were working by pure chance and not supported and download links should be fetched from `pip` which I didn't want to implement as if it's true, its a wider issue and needs to be solved in eclass.

Reply:
Never mind, wild goose chase. Its the dashes vs underscores.